### ISSUE TITLE
False positive for crispEdges in value-keyword-case

### ISSUE THREAD

**Issue by marcobiedermann** [stylelint/stylelint#1997]
> Feature request: `value-keyword-case` new option

Add "camelCase" and "except" option.

> Which rule, if any, is this issue related to?

`value-keyword-case`

> What CSS is needed to reproduce this issue?

```css
svg path {
  shape-rendering: crispEdges;
}
```

> What stylelint configuration is needed to reproduce this issue?

```json
{
  "rules": {
    "value-keyword-case": "lower"
  }
}
```

> Which version of stylelint are you using?

`7.4.2`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

Gulp Taskrunner:

```javascript
import gulp from 'gulp';
import gulpPostcss from 'gulp-postcss';
import stylelint from 'stylelint';

const dirs = {
  source: './source',
  dest  : './dist'
};

gulp.task('lint:css', () => {
  return gulp.src(`${dirs.source}/assets/css/**/*.css`)
    .pipe(gulpPostcss([
      stylelint()
    ]));
})
```

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

No

> What did you expect to happen?

"No warnings to be flagged."

> What actually happened (e.g. what warnings or errors you are getting)?

```shell
Expected "crispEdges" to be "crispedges" (value-keyword-case)
```

Here are the best ways to help resolve your issue:

```json
"value-keyword-case": ["lower", {
  "except": ["crispEdges"]
}]
```

**Comment by evilebottnawi** [comment]
@marcobiedermann thanks for feedback

**Comment by jeddy3** [comment]
@marcobiedermann Thanks for the report and for using the template.

The `value-keyword-case` rule should handle these camelCase words _by default_. I thought we'd [accounted for them all](https://github.com/stylelint/stylelint/blob/master/src/reference/keywordSets.js#L281-L286), but clearly we're missed this one.

@marcobiedermann Would you like to submit a PR adding it?

**Comment by evilebottnawi** [comment]
Done in https://github.com/stylelint/stylelint/pull/1998

**Comment by marcobiedermann** [comment]
Wow, this was quick! Thank you @evilebottnawi for your implementation 👍
